src/mpid/ch4/shm/xpmem/xpmem_avl.h

```diff
@@ -0,0 +1,564 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2006 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ *  Portions of this code were written by Intel Corporation.
+ *  Copyright (C) 2011-2017 Intel Corporation.  Intel provides this material
+ *  to Argonne National Laboratory subject to Software Grant and Corporate
+ *  Contributor License Agreement dated February 8, 2012.
+ */
+#ifndef XPMEM_AVL_H_INCLUDED
+#define XPMEM_AVL_H_INCLUDED
+
+#include "mpidimpl.h"
+#include "xpmem_pre.h"
+#include <xpmem.h>
+/*
+=== BEGIN_MPI_T_CVAR_INFO_BLOCK ===
+
+cvars:
+    - name        : MPIR_CVAR_CH4_XPMEM_AVL_TREE_BUFFER_NUM
+      category    : CH4
+      type        : int
+      default     : 64
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        Maximal number of AVL tree node buffer pool we can have.
+
+      - name      : MPIR_CVAR_CH4_XPMEM_AVL_TREE_BUFFER_NODE_NUM
+      category    : CH4
+      type        : int
+      default     : 4096
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        The number of node each node buffer pool is able to provide.
+
```

Comment:
Where do you use these environment variables ? Please delete them if it is unused.